packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -1786,7 +1787,7 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
       const runTestsInput = new V1_RunTestsInput();
       runTestsInput.model = this.getFullGraphModelData(graph);
       const runTestableInput = new V1_RunTestsTestableInput();
-      const unitAtomicTest = new V1_AtomicTestId();
+      const unitAtomicTest = new V1_UniqueTestId();
```

Comment:
did we double check that generate expected result and generate test data is working as expected with this change?

Reply:
tested generate expected result. But couldn't test generate test data as the api is not open sourced